src/pacer.js

```diff
@@ -87,6 +87,16 @@ let PACER = {
     return !!url.match(/\/(DktRpt|HistDocQry)\.pl\?\d+$/);
   },
 
+  // Returns true if the URL is for the manage account page.
+  isManageAccountPage: function(url){
+    return /pacer./.test(url) && /manage/.test(url)
+  },
+
+  // Returns true if the URL is for the iQuery page.
+  isIQueryReportUrl: function(url){
```

Comment:
Per conversation on Slack, let's rename this as `isBlankQueryReportUrl`, and then add a comment that explains the difference between the two types of URLs that this matches, and why.